mindsdb/integrations/handlers/shopify_handler/shopify_tables.py

```diff
@@ -317,6 +317,15 @@ def update_customers(self, customer_ids: List[int], values_to_update: List[Dict[
             customer.save()
             logger.info(f'Customer {customer_id} updated')
 
+    def __init__(self, handler: ShopifyHandler):
```

Comment:
This `__init__` function is not necessary here. Please see this implementation for an example: https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/shopify_handler/shopify_tables.py#L86

Reply:
Yess i have considered it and i have added some lines of code to make it work hence added a commit to solve both issues.

Reply:
Thanks. Did you test it after making these changes?

Reply:
Yes i had made changes in the gitpod directory and not in github files so i have done that now hence the screenshots are of earlier gitpod activation.